packages/dds/matrix/api-report/matrix.api.md

```diff
@@ -6,26 +6,24 @@
 
 import { IChannel } from '@fluidframework/datastore-definitions';
 import { IChannelAttributes } from '@fluidframework/datastore-definitions';
-import { IChannelAttributes as IChannelAttributes_2 } from '@fluidframework/datastore-definitions/internal';
 import { IChannelFactory } from '@fluidframework/datastore-definitions';
 import { IChannelServices } from '@fluidframework/datastore-definitions';
-import { IChannelStorageService } from '@fluidframework/datastore-definitions/internal';
-import { IEvent } from '@fluidframework/core-interfaces/internal';
-import { IEventProvider } from '@fluidframework/core-interfaces/internal';
-import { IEventThisPlaceHolder } from '@fluidframework/core-interfaces/internal';
+import { IChannelStorageService } from '@fluidframework/datastore-definitions';
+import { IEvent } from '@fluidframework/core-interfaces';
+import { IEventProvider } from '@fluidframework/core-interfaces';
+import { IEventThisPlaceHolder } from '@fluidframework/core-interfaces';
```

Comment:
So did these `/internals` show up somewhere along the RC2 release branch?  Or after his commit or something?  Surprised to see it, but if the checks pass it'll be fine

Reply:
I would assume they came at a later point in the branch? I kept getting errors on any that I left in, but I was surprised too.